pkg/cmd/dev/doctor.go

```diff
@@ -195,13 +196,31 @@ You can install node with: `+"`pkg install node`")
 		}
 	}
 
+	d.log.Println("doctor: running githooks check")
```

Comment:
Is this stuff supposed to be in a different PR? If not, mentioning it in the PR description would be good. :P

Reply:
Done.